src/test/java/homework8/expandtesting/BaseTestExpandtesting.java

```diff
@@ -0,0 +1,107 @@
+package homework8.expandtesting;
+
+import BaseTest.BaseTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BaseTestExpandtesting extends BaseTest {
+    final static String BASEURL = "https://practice.expandtesting.com";
```

Comment:
Please follow the naming conventions for constants
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html
![image](https://github.com/Suren76/Java/assets/98900791/b34aab3f-1633-4866-bb46-892948fbacd0)